src/linkedlist.js

```diff
@@ -0,0 +1,141 @@
+class Node {
```

Comment:
Please separate out your code in a separate PR. Let's keep this PR just for the documentation